### DISCUSSION TITLE
EQ1 reflections

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/library#244] [General]
Here are some thoughts after using EQ1 in class:

- In [Activity 1.1.4](https://library.tbil.org/2024/precalculus/instructor/EQ1.html#EQ1-3-5), a little something to connect what they get when they solve to what that means. Like if they get a statement that is always true (like 4x-4 = 4x-4, or 4x=4x or -4=-4, or all the way to 0=0), there are infinitely many solutions. And then when you get like 0=14, that's never true and there is no solution. So maybe ask about that first before asking the number of solutions. Or give them one of the infinitely many/no solution equations. a) Solve the linear equation. Which of the following is your last step? (answer choices like x=5, 0=0, 0=10 or something) b) Is that a statement that is always true, sometimes true, never true? c) What do you think this means about the number of solutions?

- The sentence at the beginning of [Activity 1.1.6](https://library.tbil.org/2024/precalculus/instructor/EQ1.html#EQ1-3-7) feels more like a remark segueing into the inequality discussion than an intro to the activity. Maybe add a remark to introduce what we are about to talk about. 

- Change [Activity 1.1.6(b)](https://library.tbil.org/2024/precalculus/instructor/EQ1.html#EQ1-3-7-3) to say "Which of these values **are** solutions of.. " to indicate there is more than one answer. Or add "select all that apply" or something.

- An extra step between [Activity 1.1.6(b)](https://library.tbil.org/2024/precalculus/instructor/EQ1.html#EQ1-3-7-3) 
and [Activity 1.1.6(c)](https://library.tbil.org/2024/precalculus/instructor/EQ1.html#EQ1-3-7-4) where they think about all the solutions. "There are more solutions to the inequality than the ones found in part b. How would you characterize all of them? answer: all numbers greater than 2 and 2 itself"

- Then maybe switch the order of [Activity 1.1.6(c)](https://library.tbil.org/2024/precalculus/instructor/EQ1.html#EQ1-3-7-4) 
and [Activity 1.1.6(d)](https://library.tbil.org/2024/precalculus/instructor/EQ1.html#EQ1-3-7-5). I've found many of my students struggle with interval notation and need the graph to get there. Especially at first.

**Comment by siwelwerd** [comment]
See #566
